### PR TITLE
Issue/206

### PR DESCRIPTION
**Pull request by collinmcneese**

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.18.0
+0.18.1
```
Comment (TheLunaticScripter) [inline VERSION]: This needs to get pulled out we don't do version bumps for individual changes at the moment.

File: scaffolding-chef-inspec/lib/run.ps1 [hunk 2]
```diff
@@ -0,0 +1,59 @@
+$env:SSL_CERT_FILE="{{pkgPathFor "scaffold_cacerts"}}/ssl/cert.pem"
+$env:SSL_CERT_DIR="{{pkgPathFor "scaffold_cacerts"}}/ssl/certs"
+$env:PATH = "{{pkgPathFor "scaffold_inspec_client"}}/bin;$env:PATH"
```
Comment (TheLunaticScripter) [inline scaffolding-chef-inspec/lib/run.ps1]: this line can get removed if the fix is adding the `pkgPathFor` to the inspec call.

File: scaffolding-chef-inspec/lib/run.ps1 [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+$env:CHEF_LICENSE = $env:CFG_CHEF_LICENSE
+$WAIVER="{{pkg.svc_config_path}}/waiver.yml"
+$CONFIG="{{pkg.svc_config_path}}/inspec_exec_config.json"
+$PROFILE_PATH="{{pkg.path}}/{{pkg.name}}-{{pkg.version}}.tar.gz"
+
+function Invoke-Inspec {
+
+    # # TODO: This is set to --json-config due to the
+    # #  version of InSpec being used please update when InSpec is updated
+    if([Version]({{pkgPathFor "scaffold_inspec_client"}}/bin/inspec --version) -gt [Version]"4.17.27"){
```
Comment (TheLunaticScripter) [inline scaffolding-chef-inspec/lib/run.ps1]: I think based off the issue this will still fail for a lower version of inspec

**Comment by collinmcneese** [comment]
Thanks for the notes!  Will do a less-touch PR instead with associated tests in place too

**Pull request closed without merging**